test/parallel/test-primordials-access.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+// Flags: --expose-primordials
+/* global primordials */
+
+const common = require('../common');
+const assert = require('assert');
+
+assert.strictEqual(typeof primordials, 'object');
+
+assert.throws(() => {
+  primordials.Date = common.mustNotCall(function Date() {
```

Comment:
nit: does this need the mustNotCall? It doesn't seem to actually be calling Date() anywhere